# Command gripper state (open/close)

string command
bool wait
---
bool result
